src/main/java/bt/dapps/SignumArt2.java

```diff
@@ -204,9 +204,8 @@ public void putForAuction(long priceNQT, long maxPrice, int timeout) {
 	 * The owner can accept the offer and the offer is cancelled/refunded if the item is
 	 * actually sold or a running auction ends.
 	 * 
-	 * @param offerNQT
 	 */
-	public void makeOffer(long offerNQT) {
+	public void makeOffer() {

```

Comment:
this would change the method signature, meaning all webpage integrations should be adjusted.

Reply:
I believe we should not make the change on this contract, only on another version.

Reply:
> this would change the method signature, meaning all webpage integrations should be adjusted.

Check master - here we have already the function without a variable within - so i think deplyoment was not adjusted ?

Reply:
Also https://github.com/signum-network/SIPs/blob/master/SIP/sip-40.md description is with the target version